metricq_source_bacnet/source.py

```diff
@@ -224,6 +231,7 @@ async def task(self):
                     if "presentValue" in object_result and isinstance(
                         object_result["presentValue"], (int, float)
                     ):
+                        self._last_value_send_by_metric[metric_id] = timestamp
```

Comment:
Shouldn't this line come after the `await send`, in case it throws an exception?